js/dom.test.js

```diff
@@ -0,0 +1,17 @@
+const { JSDOM,VirtualConsole} = require('jsdom')
+
+describe('DOM Tests', () => {
+    let dom
+    let document
+    
+    //load the DOM
+    beforeAll(async () => {
+        dom = await initDom('index.html')
+        document = dom.window.document
+    })
+
+    test(`if spielfeld exists`, () => {
```

Comment:
Für den einfachen Test reicht es noch die test-funktion aufzurufen. Aber sobald mehr interaktion in der Seite ist, wirst Du mit itWill arbeiten müssen.

Versuch vielleicht nochmal ein paar Tests  zu schreiben. Z.B. ob sich die position verändert hat, wenn ein key gedrückt wird.